design-system/gulpfile.js

```diff
@@ -44,15 +38,6 @@ const dest_paths = {
 const settings = {
   css: {
     outputStyle: 'compressed',
-    includePaths: [
```

Comment:
We didn't need these as we can specify the paths in the CSS instead (and symlinks work!)

**Before**
```css
/* Foundations */
@import "normalize.css/normalize.css";
@import "foundations-vars/src/vars.pcss";
@import "foundations-global/src/global.pcss";
```

**After**
```css
/* Foundations */
@import "normalize.css";
@import "@coopdigital/foundations-vars";
@import "@coopdigital/foundations-global";
```